tests/valgrind/potential-memleaks.c

```diff
@@ -9,6 +9,15 @@
 #include <stdlib.h>
 #include <string.h>
 
+#if defined(__FreeBSD__)
+/*
+ * We only want to enable this check on FreeBSD, because that OS can use
+ * pthread_atfork() without explicitly linking to pthread.
```

Comment:
I assume you know how FreeBSD does this.

Reply:
Sure, pthread_atfork is part of libc. ;-)

Or at least a stub function is.  For same same reasons as we're (umm, once I get around to reviewing it) going to have "maybelock".